src/Avalonia.Styling/StyledElement.cs

```diff
@@ -16,6 +16,24 @@
 
 namespace Avalonia
 {
+    public class StyleDiagnostics
```

Comment:
Should probably move these to the `Avalonia.Diagnostics` namespace.

Reply:
Yep, done.